src/get-started/production/index.md.njk

```diff
@@ -64,31 +64,13 @@ Explore the [Styles](../../styles/) section of the Design System to see what cla
 
 ## Using components
 
-The components in the Design System are designed to be accessible and responsive - there are 2 ways to implement them in your application.
+The components in the Design System are designed to be accessible and responsive.
+
+You can use them in your live application as either:
 
-You can either use HTML or - if you’re using Nunjucks with node.js and you installed GOV.UK Frontend using npm - you can use a Nunjucks Macro.
+- HTML
```

Comment:
This might be something to look at later, but I think we should be doing more to encourage people to have some sort of abstraction layer for their components, which might mean:

- using an existing resource maintained by someone else (so linking to the community resources section)
- helping them understand why they might want to make their own, and what the benefits are

If that is out of scope for this PR, I wonder if there's a way to soften this so it doesn't make it sound like those are the only two options?

Reply:
Happy to look at this next sprint potentially? I think it might be best to leave as is for now though, to avoid the scope from growing too much.